core/instance.go

```diff
@@ -212,7 +212,9 @@ func (i *instance) isStorageCompatible(spotCandidate instanceTypeInformation, at
 
 func (i *instance) isVirtualizationCompatible(spotVirtualizationTypes []string) bool {
 	current := *i.VirtualizationType
-
+	if len(spotVirtualizationTypes) == 0 {
+		spotVirtualizationTypes = append(spotVirtualizationTypes, "HVM")
```

Comment:
I think `[]string{"HVM"}` would be better than the append

Reply:
@cristim done